src/lib/browse/layers/CrossingsLayerControl.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+  import { circleRadius } from "colors";
+  import { ExternalLink } from "lib/common";
+  import type { MapGeoJSONFeature } from "maplibre-gl";
+  import crossingsUrl from "../../../../assets/crossings.geojson?url";
+  import PointAmenityLayerControl from "./PointAmenityLayerControl.svelte";
+
+  const name = "crossings";
+
+  function tooltip(feature: MapGeoJSONFeature): string {
+    const crossingTypeString: string = feature.properties["crossing"];
+    if (crossingTypeString === "no")
+      return "Location where crossing is impossible/illegal but where there is a clear desire line to cross.";
+    let result = "Crossing with the following features.";
+    const initialTooltipSentence =
+      initialTooltipSentences.get(crossingTypeString);
+    if (initialTooltipSentence) result = initialTooltipSentence;
+
+    return result + `. More information can be found by clicking.`;
```

Comment:
We've been using `${result}. Foo bar xyz` for string interpolation, not concat generally, in this codebase